src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -2430,22 +2422,22 @@ class C
             End Using
         End Function
 
-        <WorkItem(930254, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/930254")>
-        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
-        Public Async Function NoCompletionWithBoxSelection() As Task
-            Using state = TestState.CreateCSharpTestState(
-                <Document><![CDATA[
-class C
-{
-    {|Selection:$$int x;|}
-    {|Selection:int y;|}
-}]]></Document>)
-                state.SendInvokeCompletionList()
-                Await state.AssertNoCompletionSession()
-                state.SendTypeChars("goo")
-                Await state.AssertNoCompletionSession()
-            End Using
-        End Function
+        '        <WorkItem(930254, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/930254")>
```

Comment:
❔ Why commented out? Can we make this a skipped test that compiles and links to an issue?

Reply:
Thank you!
uncommented, skipped and covered with
[https://github.com/dotnet/roslyn/issues/29110](https://github.com/dotnet/roslyn/issues/29110)

---
In reply to: [208000192](https://github.com/dotnet/roslyn/pull/29016#discussion_r208000192) [](ancestors = 208000192)